### PR TITLE
Homework3: Danila Zhogol

### PR DESCRIPTION
**Pull request by danzhog**
The third homework for IT_Academy/Python_course

### CODE REVIEW

File: src/homework3/1_FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# FizzBuzz
+# Напишите программу, которая печатает цифры от 1 до 100,
+# но вместо чисел, кратных 3 пишет Fizz,
+# вместо чисел кратный 5 пишет Buzz,
+# а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+
+for number in range(1, 101):
+    if number % 3 == 0 and number % 5 == 0:
```
Comment (belovmd) [inline src/homework3/1_FizzBuzz.py]: 15?

File: src/homework3/2_list_practice.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# Используйте на предыдущий список slice чтобы получить следующий:
+# ['ab', 'ad', 'bc'].
+
+print(lst[::2])
+
+# Используйте генератор списков чтобы получить следующий:
+# ['1a', '2a', '3a', '4a'].
+
+lst = [(str(num) + 'a') for num in range(1, 5)]
```
Comment (belovmd) [inline src/homework3/2_list_practice.py]: '1234' ?

**Review by belovmd — COMMENTED** [review]
1. Please don't remove init.py
2. Please address these issues

File: src/homework3/1_FizzBuzz.py (hunk 1 above)
Comment (danzhog) [reply src/homework3/1_FizzBuzz.py]: oh yeeah...))

File: src/homework3/2_list_practice.py (hunk 2 above)
Comment (danzhog) [reply src/homework3/2_list_practice.py]: ++ done.

**Review by belovmd — COMMENTED** [review]
1) __init__.py still deleted. 
8,5

**Pull request closed without merging**